api/health_check.go

```diff
@@ -64,11 +65,14 @@ func (app *ApiServer) healthCheck(c *fiber.Ctx) error {
 
 	healthyNodes := app.contentNodeMonitor.GetContentNodes()
 	// Convert config.Node to contentNode
-	contentNodes := make([]contentNode, len(healthyNodes))
-	for i, node := range healthyNodes {
-		contentNodes[i] = contentNode{
-			DelegateOwnerWallet: node.DelegateOwnerWallet,
-			Endpoint:            node.Endpoint,
+	contentNodes := make([]contentNode, 0, len(healthyNodes))
+	for _, node := range healthyNodes {
+		// icky reaching into config to check upload nodes
+		if slices.Contains(app.contentNodeMonitor.config.UploadNodes, node.Endpoint) {
```

Comment:
Can't we just read Config directly here? Not sure why we need to grab it from the monitor.